translate-pixiv-tags.user.js

```diff
@@ -319,8 +324,106 @@ const PROGRAM_CSS = `
 }
 `;
 
+const CACHE_PARAM = (CACHE_LIFETIME ? { expires_in: CACHE_LIFETIME } : {});
+// Setting this to the maximum since batches could return more than the amount being queried
+const LIMIT_PARAM = { limit: 1000 };
+
+function WikiParams (otherNamesList) {
```

Comment:
camelCase but not PascalCase.
Also, why not include them into the NETWORK_REQUEST_DICT? And *_FIELDS as well.

Reply:
Ah, sorry, I wasn't paying attention on the caseness there, and the linter didn't complain at me either. Is that a rule that needs to be set in the linter config so that that doesn't happen again in the future?

Anyways, I tried including the functions in the hash originally, but the linter complained at me with the following error:

`error    Expected method shorthand`

I guess I gave up too early because I figured out how to do it just now.

Also, I didn't think about moving the fields to that hash, but that's a good idea so I'll go ahead and do that as well.